timer/scheduler.go

```diff
@@ -0,0 +1,63 @@
+package timer
+
+import (
+	"time"
+
+	"github.com/nivista/steady/.gen/protos/common"
+	rcron "github.com/robfig/cron/v3"
+)
+
+type cron struct {
+	sched               rcron.Schedule
+	startTime, stopTime time.Time
+	maxExecutions       int
+}
+
+var parser = rcron.NewParser(rcron.Minute | rcron.Hour | rcron.Dom | rcron.Month | rcron.Dow | rcron.SecondOptional | rcron.Descriptor)
+
+// infiniteExecutions indicates a timer should fire until cancelled.
+const infiniteExecutions = 0
```

Comment:
I thought we were going to define this in proto as a constant? This is a weird customer experience if they don't have a useful constant here.

Reply:
Ok they do have this constant. I had it twice. I'll remove this and use the one from protos.